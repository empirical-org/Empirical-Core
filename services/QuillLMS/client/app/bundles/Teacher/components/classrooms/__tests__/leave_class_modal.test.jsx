import { shallow } from 'enzyme';
import React from 'react';

import LeaveClassModal from '../leave_class_modal';

import { classroomWithStudents } from './test_data/test_data';

describe('LeaveClassModal component', () => {

  const wrapper = shallow(
    <LeaveClassModal
      classroom={classroomWithStudents}
      close={() => {}}
      onSuccess={() => {}}
    />
  );

  it('should render LeaveClassModal', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
