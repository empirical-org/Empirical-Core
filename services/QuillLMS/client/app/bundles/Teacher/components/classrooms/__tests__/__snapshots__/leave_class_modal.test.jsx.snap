// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeaveClassModal component should render LeaveClassModal 1`] = `
<div
  className="modal-container leave-class-modal-container"
>
  <div
    className="modal-background"
  />
  <div
    className="leave-class-modal quill-modal modal-body"
  >
    <div>
      <h3
        className="title"
      >
        Leave class?
      </h3>
    </div>
    <div
      className="leave-class-modal-text"
    >
      <p>
        You won't be able to view or manage this class unless you're re-invited.
      </p>
    </div>
    <div
      className="form-buttons"
    >
      <button
        className="quill-button-archived outlined secondary medium"
        onClick={[Function]}
      >
        Cancel
      </button>
      <button
        className="quill-button-archived contained primary medium"
        onClick={[Function]}
      >
        Leave class
      </button>
    </div>
  </div>
</div>
`;
